src/pynwb/data/UpdateFromUpstream.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+thirdparty_module_name='nwb-schema'
+
+upstream_git_url='git://github.com/NeurodataWithoutBorders/nwb-schema'
+upstream_git_branch='dev'
+github_compare=true
+
+snapshot_author_name='nwb-schema Upstream'
+snapshot_author_email='neurodatawithoutborders@googlegroups.com'
```

Comment:
@oruebel I suggest we associated the email `neurodatawithoutborders@googlegroups.com` with @nwb-bot

Reply:
@oruebel If this sounds reasonable, is it something you could do ?

Reply:
@jcfr what is the purpose of linking nwb-bot to the googlegroup?

Reply:
This allows to have commit message done by the bot to have an email, allowing it to link back to the bot account.

The email doesn't have to be verified, it should just be added in the settings.